sdks/java/core/src/main/java/org/apache/beam/sdk/coders/StringDelegateCoder.java

```diff
@@ -17,9 +17,14 @@
  */
 package org.apache.beam.sdk.coders;
 
+import org.apache.beam.sdk.coders.DelegateCoder.CodingFunction;
```

Comment:
I was hoping we'd remove StringDelegate altogether, inlining it in the one place (TFIDF example) that it's used.